scripts/test_context_broker.sh

```diff
@@ -8,12 +8,22 @@ jq '.access_token'`
 export token="${token%\"}"
 export token="${token#\"}"
 
-echo "\n\ncan i read entities?"
-curl -i -H "Authorization: Bearer $token" -H "fiware-service: Tenant1" -H "fiware-servicepath: /" 'http://localhost:8000/v2/entities'
+echo "Can i read entities?"
+echo "==============================================================="
+export response=`curl -s -o /dev/null -w "%{http_code}" -H "Authorization: Bearer $token" -H "fiware-service: Tenant1" -H "fiware-servicepath: /" 'http://localhost:8000/v2/entities'`
+if [ $response == "200" ]
+then
+  echo "PASSED"
+else
+  echo "ERROR: Can't read entities"
+  exit 1
```

Comment:
shouldn't be -1?

Reply:
forgive me ignorance on this aspects :)

Reply:
Exiting with 1 indicates a general error. There are [other errors too](https://shapeshed.com/unix-exit-codes/), but I don't think -1 is an error.